README.md

```diff
@@ -116,6 +116,7 @@ of releases [here](https://github.com/tensorflow/io/releases).
 
 | TensorFlow I/O Version | TensorFlow Compatibility | Release Date |
 | --- | --- | --- |
+| 0.17.1 | 2.4.x | Apr 15, 2021 |
```

Comment:
This should be April 16?

Reply:
@terrytangyuan Updated. Please take a look.